frontend/stubs/list_methods.py

```diff
@@ -0,0 +1,52 @@
+"""Stub file for methods invoked on lists"""
+from typing import TypeVar, List
+
+T = TypeVar("T")
+
+
+def append(l: List[T], e: T) -> None:
```

Comment:
Do we have any way of distinguishing if something like append is a method of the class list, or if it is a top level function that just takes a list as its first argument?